FeedService/Functions/GetPiletFile.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Net.Http;
+using FeedService.Services;
+
+namespace FeedService.Functions
+{
+    public  class GetPiletFile
+    {
+        private readonly IPiletService _piletService;
+        public GetPiletFile(IPiletService piletService)
+        {
+            _piletService = piletService;
+        }
+
+        [FunctionName("GetPiletFile")]
+        public async Task<HttpResponseMessage> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "files/{name}/{version}/{fileName}")] HttpRequestMessage req,
+            ILogger log, string name, string version, string fileName)
+        {
+            var file = await _piletService.GetPiletFile(fileName, version, name);
+            var response = req.CreateResponse(System.Net.HttpStatusCode.OK);
+            response.Content = new StringContent(file, System.Text.Encoding.UTF8, "text/css");
```

Comment:
Is this always of type `text/css`? Also if a `.js` file is wanted here? What about binary files (e.g., WASM)? Should we return `StringContent` or actually get the file's content as a `MemoryStream` which is then put as a generic content response?

Reply:
What i have test with this is it returns all type of files. i.e. .js or .css. I was able to retrieve the pilet successfully.

Reply:
Yes sure it always *returns*, but the question is if the browser will actually *accept* it. If its the wrong mime type certain assets will be rejected by the browser. Its therefore quite crucial to set the right mime type.

Reply:
Shall i  do like this. 
For css i return (text.css)
for JS i return (text/javascript)

Reply:
I'd just detect the mime type via the extension. Potential solution: https://stackoverflow.com/a/14108040/1306682